airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/StateBlock.tsx

```diff
@@ -28,3 +28,16 @@ export const StateBlock: React.FC<StateBlockProps> = ({ connectionId }) => {
     </Card>
   );
 };
+
+function displayState(
```

Comment:
I think this file is more readable if we move `displayState()` above `StateBlock()`.